modules/ROOT/pages/_partials/push/registering_device.adoc

```diff
@@ -105,6 +106,39 @@ NOTE: Optionally, you can pass the parameters below to the register method
   timeout: 5000 // in milliseconds
 }
 ----
+
+WARNING: It is very important that you provide the correct path to your service worker: it will be responsible of
```

Comment:
```suggestion
WARNING: It is very important that you provide the correct path to your service worker: it will be responsible for
```

Reply:
I'd suggest we try this rewrite again.

Reply:
I'd suggest removing this note, where can you provide an incorrect path in any software?

Reply:
I have fixed it directly on WebPush branch. See: https://github.com/aerogear/mobile-docs/commit/695d240578182ebc57f3f71d55c1c06608ac84b0